openebs/maya
# Merge openebs-exporter with maya-agent

Merge openebs-exporter with maya-agent
    
**What this PR does**:
    
* This PR refers to issue [#683](https://github.com/openebs/openebs/issues/683)
* This PR merge openebs-exporter with maya-agent
    
**why we need it**:
    
* maya-agent will be used for monitoring purpose.
    
**How this commit address the issue**:
    
* a command `monitor` is added along with some flags which can be used to
      monitor volumes.e.g. `maya-agent monitor -c=http://localhost:9501`
    
**How to verify this change**
    
* run this binary locally or in maya-apiserver container and do
  `curl localhost:9500/metrics` and make sure volume is being
   consumed otherwise it will show zero in all the stats fields.

## Note    
* An [issue](https://github.com/openebs/openebs/issues/690) is created to write buildscripts and Dockerfile.
